cns/azure-cns.yaml

```diff
@@ -96,7 +96,7 @@ spec:
           effect: NoSchedule
       containers:
         - name: cns-container
-          image: mcr.microsoft.com/containernetworking/azure-cns:v1.4.7
+          image: mcr.microsoft.com/containernetworking/azure-cns:v1.4.29
```

Comment:
Can you revert this? I think this manifest is used by partners, and is unrelated to this change anyway